src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -298,14 +298,14 @@ class BankAccountStep extends React.Component {
                                     </View>
                                 )}
                             />
+                            <Button
+                                success
+                                text={this.props.translate('common.saveAndContinue')}
+                                style={[styles.mb4]}
+                                isDisabled={!this.canSubmitManually()}
+                                onPress={this.addManualAccount}
+                            />
                         </View>
```

Comment:
I know there's not a lot going on in this step - but I think this should be a `ScrollView`?

Reply:
Replaced View with ScrollView as per request, also adjusted style accordingly.